2-Regression/translations/README.ja.md

```diff
@@ -0,0 +1,32 @@
+# 機械学習のための回帰モデル
+## トピック: 北米のカボチャ価格に関する回帰モデル 🎃
+
+北米では、ハロウィンのためにカボチャはよく怖い顔に彫られています。そんな魅力的な野菜についてもっと知りましょう！
+
+![jack-o-lanterns](../images/jack-o-lanterns.jpg)
+> <a href="https://unsplash.com/@teutschmann?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Beth Teutschmann</a>によって<a href="https://unsplash.com/s/photos/jack-o-lanterns?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>に投稿された写真
+
+## 今回学ぶこと
+この章のレッスンでは、機械学習の文脈における回帰の種類について説明します。回帰モデルは変数間の"関係"を決定するのに役立ちます。このタイプのモデルは、長さ、温度、年齢などの値を予測し、データポイントの分析をすることで変数間の関係性を明らかにします。
+
+今回のレッスンでは、線形回帰とロジスティック回帰の違いやどのように使い分けるかを説明します。
+
+データサイエンティストの共通開発環境であるノートブックを管理するためのVisual Studio Codeの構成や機械学習のタスクを開始するための準備を行います。また、機械学習用のライブラリであるScikit-learnを利用し最初のモデルを構築します。この章では回帰モデルに焦点を当てます。
+
+> 回帰モデルを学習するのに役立つローコードツールがあります。ぜひ[Azure ML for this task](https://docs.microsoft.com/learn/modules/create-regression-model-azure-machine-learning-designer/?WT.mc_id=academic-15963-cxa)を使ってみてください。
+
+### レッスン
+
+1. [商売道具](../1-Tools/README.md)
```

Comment:
hi, you're going to want to change these paths to something like ../1-Tools/translations/README.ja.md eventually, so perhaps add this translation to a draft PR and commit them all when ready? thank you!

Reply:
Thank you for advice. This PR change to draft until to finish translate related lesson. I added checkbox on PR comment.